src/CoreComputing.jl

```diff
@@ -158,8 +158,8 @@ function map_pairwise_serial!(
     show_progress::Bool=false
 ) where {F,N,T}
     p = show_progress ? Progress(length(cl.ref), dt=1) : nothing
-    for i in eachindex(cl.ref)
-        output = inner_loop!(f, output, i, box, cl)
+    @batch for i in eachindex(cl.ref)
```

Comment:
This insertion of ˋ@batchˋ is in the serial implementation, so I´m unsure if that was what you were willing to do.

Reply:
That was the easiest way to try this out.

Reply:
In a few tests I don't see the difference in performance you report. I might have to dig into your example to see. 

One thing that might be important (I didn't look closely to your example) is the density of particles. My tests have the `10^5/(100^3)` density, which is the atomic density of water, in the range of the applications I'm interested in. 

The tradeoffs in other density ranges are different.

Reply:
How is the density relevant? Because of the cutoff? I used a cutoff of 3 times the spacing of a grid of particles. For more realistic benchmarks, I randomly perturbed the particle positions.

Reply:
Yes, of course, in relation to the cutoff. A typical molecular dynamics simulation has a density of ˋ10ˆ5/100ˆ3ˋ particles, and use a cutoff of something of around 8 to 15, in the same units.

In some analsys of galaxy distributions (not my speciality), people use densities of ˋ10ˆ5/20.274ˆ3ˋ for a cutoff of 5.0. (much denser systems).

These define how many particles end up in each cell, and then what is worth doing in the inner loops that compute the interactions inside each cell or among neighbor cells. One parameter is the side of the cell relative to the cutoff (ˋlcellˋ in CellListMap), for very dense systems a cell side smalller than thte cutoff is beneficial. An in CellListMap y also project the particles in the vector connecting cell centers and then sort the projected distances to avoid even further computing spurious distances.